awx/main/utils/common.py

```diff
@@ -581,7 +581,6 @@ def prefetch_page_capabilities(model, page, prefetch_list, user):
         6: {'edit': False, 'start': False}
     }
     Each capability is produced for all items in the page in a single query
-
```

Comment:
I know it seems super nit-picky, but can you remove the change that deletes these two lines? I still have to write tests for your main changes, but in the meantime we try to keep our changes as small as possible (even if they are just removing blank lines). :+1: